docs/ImplicitMSALAuthenticationProvider.md

```diff
@@ -0,0 +1,56 @@
+#### Creating an instance of ImplicitMSALAuthenticationProvider in browser environment
+
+Refer devDependencies in [package.json](../package.json) for the compatible msal version and update that version in below.
+
+**Important Note:** MSAL is supported only for frontend applications, for server-side authentication you have to can use [TokenCredentialAuthenticationProvider](./TokenCredentialAuthenticationProvider.md) or implement your own AuthenticationProvider. Learn how you can create a [Custom Authentication Provider](./docs/CustomAuthenticationProvider.md).
+
+```html
+<script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.min.js"></script>
+```
+
+```typescript
+// Configuration options for MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/MSAL.js-1.0.0-api-release#configuration-options
+const msalConfig = {
+	auth: {
+		clientId: "your_client_id", // Client Id of the registered application
+		redirectUri: "your_redirect_uri",
+	},
+};
+const graphScopes = ["user.read", "mail.send"]; // An array of graph scopes
+
+// Important Note: This library implements loginPopup and acquireTokenPopup flow, remember this while initializing the msal
+// Initialize the MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js#1-instantiate-the-useragentapplication
+const msalApplication = new Msal.UserAgentApplication(msalConfig);
+const options = new MicrosoftGraph.MSALAuthenticationProviderOptions(graphScopes);
+const authProvider = new MicrosoftGraph.ImplicitMSALAuthenticationProvider(msalApplication, options);
```

Comment:
this looks like the option for how it would be done in javascript not typescript.  Should we show somewhere how to import the implicitmsalauthenticationprovider in typescript?

Reply:
There is typescript example in this documentation.